explainer.md

```diff
@@ -6,58 +6,191 @@
+    response = await fetch(weightUrl);
+    buffer = await response.arrayBuffer();
+    const weightData117 = new Float32Array(buffer, 0, WEIGHT_SIZE);
+    const weightData118 = new Float32Array(buffer, WEIGHT_SIZE, WEIGHT_SIZE);
+    const weightData137 = new Float32Array(buffer, 2 * WEIGHT_SIZE, WEIGHT_SIZE);
+    const weightData138 = new Float32Array(buffer, 3 * WEIGHT_SIZE, WEIGHT_SIZE);
+    const weightData157 = new Float32Array(buffer, 4 * WEIGHT_SIZE, WEIGHT_SIZE);
+    const weightData158 = new Float32Array(buffer, 5 * WEIGHT_SIZE, WEIGHT_SIZE);
+    response = await fetch(biasUrl);
+    buffer = await response.arrayBuffer();
+    const biasData119 = new Float32Array(buffer, 0, BIAS_SIZE);
+    const biasData139 = new Float32Array(buffer, BIAS_SIZE, BIAS_SIZE);
+    const biasData159 = new Float32Array(buffer, 2 * BIAS_SIZE, BIAS_SIZE);
+    response = await fetch(initializerUrl);
+    buffer = await response.arrayBuffer();
+    const initData160 = new Float32Array(buffer, 0, MATMUL96_INIT_SIZE);
+    const initData170 = new Float32Array(buffer, MATMUL96_INIT_SIZE, ADD97_INIT_SIZE);
+    // Create constant operands
+    const builder = navigator.ml.getNeuralNetworkContext().creatModelBuilder();
+    const hidden1 = builder.constant({ type: 'float32', dimensions: HIDDEN_DIMS }, hiddenData1);
+    const hidden2 = builder.constant({ type: 'float32', dimensions: HIDDEN_DIMS }, hiddenData2);
+    const hidden3 = builder.constant({ type: 'float32', dimensions: HIDDEN_DIMS }, hiddenData3);
+    const weight117 = builder.constant({ type: 'float32', dimensions: WEIGHT_DIMS }, weightData117);
+    const weight118 = builder.constant({ type: 'float32', dimensions: WEIGHT_DIMS }, weightData118);
+    const weight137 = builder.constant({ type: 'float32', dimensions: WEIGHT_DIMS }, weightData137);
+    const weight138 = builder.constant({ type: 'float32', dimensions: WEIGHT_DIMS }, weightData138);
+    const weight157 = builder.constant({ type: 'float32', dimensions: WEIGHT_DIMS }, weightData157);
+    const weight158 = builder.constant({ type: 'float32', dimensions: WEIGHT_DIMS }, weightData158);
+    const bias119 = builder.constant({ type: 'float32', dimensions: BIAS_DIMS }, biasData119);
+    const bias139 = builder.constant({ type: 'float32', dimensions: BIAS_DIMS }, biasData139);
+    const bias159 = builder.constant({ type: 'float32', dimensions: BIAS_DIMS }, biasData159);
+    const init160 = builder.constant({ type: 'float32', dimensions: MATMUL96_INIT_DIMS }, initData160);
+    const init170 = builder.constant({ type: 'float32', dimensions: ADD97_INIT_DIMS }, initData170);
+    // Build up the network
+    const input = builder.input('input', { type:'float32', dimensions:INPUT_DIMS });
+    const [gru43, gru42] = builder.gru(input, weight117, weight118, STEP_COUNT, 257, 
+                                       bias119, null, hidden1, true, true);
+    const add45 = builder.add(input, builder.squeeze(gru42, 1));
```

Comment:
WebNN [squeeze](https://webmachinelearning.github.io/webnn/#api-modelbuilder-squeeze) requires `axes` to be `sequence<long>`. It would be 
```js
builder.squeeze(gru42, [1])
```

Reply:
good catch!

Reply:
Fixed